daemon/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	go clean
 
 install:
-	groupadd -f cilium
+	if [ ! -f /etc/redhat-release ]; then groupadd -f cilium; fi
```

Comment:
Building packages with _mockbuild_ disallow usage of tools such as `groupadd` in `make install` target. Adding groups/users should be done during package installation and declared in `%pre` section of _spec_ file.